web/yarn.lock

```diff
@@ -4,17 +4,17 @@
 
 "@aashutoshrathi/word-wrap@^1.2.3":
   version "1.2.6"
-  resolved "https://registry.yarnpkg.com/@aashutoshrathi/word-wrap/-/word-wrap-1.2.6.tgz#bd9154aec9983f77b3a034ecaa015c2e4201f6cf"
```

Comment:
doesn't feel like needed changes either for the feature ("migration from yarnpkg.com to npmjs.org")?

Reply:
This also seems like it might belong in .gitingore

Reply:
oh, I see, this was created when I accidentally ran nmp install instead of yarn install

Reply:
We can't put this it .gitignore because there are times when indeed we need to track/commit changes to this file.

Reply:
ah ok